hepdata/modules/theme/templates/hepdata_theme/page.html

```diff
@@ -78,7 +78,21 @@
 {%- endblock css %}
 
 {%- block page_header %}
-    {% include "hepdata_theme/header.html" %}
+  <div class="sticky-bars">
+
+    {% if banner_msg is not none %}
```

Comment:
Maybe change this to just `{% if banner_msg %}` so that an empty string is not displayed.